src/xdocs/google_style.xml

```diff
@@ -2437,6 +2437,30 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
+          <br/>
+          <br/>
+          <code>org.checkstyle.google.severity=${SeverityLevel}</code>
+        </p>
+        <p>
+          Visit here for more details about Severity Levels:
+          <a href="https://checkstyle.org/property_types.html#SeverityLevel">
+            Severity Level
+          </a>
+        </p>
+        <p>
+          Custom properties can be set by property file ( <code>-p</code> ) in
+          <a href="https://checkstyle.org/cmdline.html#Command_line_usage">
+            CLI
+          </a>,
+          <a href="https://checkstyle.org/anttask.html#Parameters">
+            properties parameter
+          </a>
+           in Ant task, or as system variable.
```

Comment:
> system variable

Please remove it or prove it works by CLI.
In some other thread you shared that system variables are not working properly

Reply:
In discord you suggested to open a new issue about it. Should I link that issue here and specify that system variables aren't working properly?

Reply:
done. Also opened issue about system variable problem: https://github.com/checkstyle/checkstyle/issues/15089

Reply:
Please restore it. It works .